apps/src/templates/BaseDialog.jsx

```diff
@@ -191,9 +191,22 @@ export default class BaseDialog extends React.Component {
 
     return (
       <div className={wrapperClassNames}>
-        <div className={modalBackdropClassNames} onClick={this.closeDialog} />
+        <div
+          className={modalBackdropClassNames}
+          style={styles.modalBackdrop}
```

Comment:
i'm not sure i understand this change -- what do these styles do?

Reply:
The modal applies a transparent screen to the parent behind the modal, in this case, the Animation Picker Dialog. Since the dialog has the "X" button that sticks out, I extend the size of the background by 15 pixels in all directions to cover that. Would a comment here help make sense  of this?

Reply:
i don't think this is doing what we want it to. i pulled the branch and these styles are making it look like there's a black box around the main "Animation Library" modal (maybe because the backdrops are overlapping? i'm not sure)

<img width="1440" alt="Screen Shot 2021-11-08 at 11 37 19 AM" src="https://user-images.githubusercontent.com/9812299/140807329-4e289a01-9930-4a24-bbef-afa96091dee6.png">